### PR TITLE
Add ruleset: wind.it

### PR DESCRIPTION
**Pull request by scop**

### CODE REVIEW

File: src/chrome/content/rules/Wind.it.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<ruleset name="Wind.it">
+	<target host="wind.it" />
+	<target host="www.wind.it" />
```
Comment (J0WI) [inline src/chrome/content/rules/Wind.it.xml]: Please see also:
```
start.wind.it
telefonino.wind.it
startpre.wind.it
wep.wind.it
windcloud.wind.it
windsmartcontrol.wind.it
web.pos.wind.it
authproxy.wind.it
windecare.wind.it
areaclientiaziende.wind.it:8185
windecare.wind.it
partnerportal.wind.it:4445
weg.wind.it
mdm.wind.it

Comment (scop) [reply src/chrome/content/rules/Wind.it.xml]: More thorough investigation/listing takes more time than what I have available for this, so marked as partial instead.

Comment (J0WI) [reply src/chrome/content/rules/Wind.it.xml]: Since I already searched them for you I would be happy if this work wasn't a waste of time.

Comment (scop) [reply src/chrome/content/rules/Wind.it.xml]: Added mdm and start; for the others I'm not getting http results that I believe would warrant additions.

Comment (J0WI) [reply src/chrome/content/rules/Wind.it.xml]: Thanks

**Pull request merged**